Makefile

```diff
@@ -5,7 +5,7 @@ GOGCFLAGS ?= -gcflags=all="-N -l"
 endif
 
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
-ENVTEST_K8S_VERSION = 1.31
+ENVTEST_K8S_VERSION = 1.32.1
```

Comment:
You'll also need to update the `unit` target to use `--index` similarly to https://github.com/openshift/machine-api-operator/blob/b657d8bf9389ca0047b1fc9a316dc88ee84c5311/Makefile#L106

Reply:
yep i'm just putting that in now :)